### ISSUE TITLE
Pip install errors....

### ISSUE THREAD

**Issue by Eve00000** [acolombier/stemgen#6]
### Bug Description

Hi Antoine, so I wanted to give your stemgen a shot :-)

pip install gave errors:
[20240724 Antoine Stemgen install.txt](https://github.com/user-attachments/files/16361115/20240724.Antoine.Stemgen.install.txt)

so I  wanted to give the dockerthing a shot. I have about 10 dockers running on my NAS, but haven't installed docker on windows. running your docker gives a failure.
On my NAS I could create a nice config file where my network, map bindings etc are listed, can't find it on windows.
I can't get the env right I suppose.
Can you write me a line to convert A:\Stem\ToDO\Gloria.m4a ?
I tried  with / and \ and // and \\ 
docker run -v /A:/Stem:/A:/Stem -it --rm aclmb/stemgen:0.2.0 \ /A:/Stem/ToDo/Gloria.M4A

Thanks

### Version

docker 0.2.0

### OS

win 10

**Comment by acolombier** [comment]
Hey Evelynne, thanks for giving that a go!

Likely, you won't be able to install the pip package because it requires you to have the TagLib v2 installed on your system, and that can be a nightmare on Windows machine. Going forward, if any Windows dev read this, I'd be happy to get contributions in to have taglib installed by the pip installer, but likely not something I'm going to do right away as I'm not using Windows enough to know the correct way.

When it comes to running the Docker container, same again here, Docker and Windows is pretty upsetting. Make sure you use Docker in Linux mode or WSL. 

Could you please share the error message you are getting ? I believe you should be able to do something like this:

```
docker run -v '/A/Stem:/A/Stem' -it --rm aclmb/stemgen:0.2.0 '/A/Stem/ToDo/Gloria.M4A' /A/Stem'
```
Note you might want to try following as well if you have a NVidia GPU

```
docker run --gpu -v '/A/Stem:/A/Stem' -it --rm aclmb/stemgen:0.2.0 '/A/Stem/ToDo/Gloria.M4A' /A/Stem'
```

Let me know if you get any errors!

**Comment by Eve00000** [comment]
Bonjour,
The taglib is used to copy the metadata drom the original file to the stem, right?
That is your perfectional instinct, but maybe that's overkill and not direct needed to create the stems.
Since years there's a great tool to edit/copy/clone ... metadata: [MP3Tag](https://docs.mp3tag.de/), the nice thing is the import/export part (fi export the title of a list of files to txt, edit and re-import) and the flexibility (all kinds of music files).
It's possitble to copy 'all' tafs from one file to another (just ctrl-c ctrl-v) and there is the scripting function.
So I think the metadata part isn't that important. 
I will do some extra tests without the pip install.

**Comment by acolombier** [comment]
I agree it would be great to have it optional, but unfortunately, it isn't that simple with Python.

> that's overkill and not direct needed to create the stems

The way I use this tool is to batch convert many files (100+) into stems, so having to re-tag them individually and manually, including adding artwork isn't that user friendly and not quite realistic. 

> I will do some extra tests without the pip install.

Thanks! If I get the time, I will also give that a go in a Windows VM
